### ISSUE TITLE
remove taskcluster timestamps in log viewer

### ISSUE THREAD

**Issue by froydnj** [taskcluster/unified-logviewer#37]
(Originally filed as [bug 1355810](https://bugzilla.mozilla.org/show_bug.cgi?id=1355810); repeating the rationale here.)

It's super annoying to copy and paste sections of logfiles from the log viewer and wind up with:

```
[task 2017-04-12T11:57:04.335910Z] 11:57:04     INFO - Hit MOZ_CRASH(gfxUserFontSet not thread-safe) at /home/worker/workspace/build/src/gfx/thebes/gfxUserFontSet.h:178
[task 2017-04-12T11:57:04.336831Z] 11:57:04     INFO - #01: gfxFontGroup::gfxFontGroup [gfx/thebes/gfxTextRun.cpp:1713]
[task 2017-04-12T11:57:04.337553Z] 11:57:04     INFO - 
[task 2017-04-12T11:57:04.338807Z] 11:57:04     INFO - #02: gfxPlatformGtk::CreateFontGroup [gfx/thebes/gfxPlatformGtk.cpp:281]
[task 2017-04-12T11:57:04.340467Z] 11:57:04     INFO - 
[task 2017-04-12T11:57:04.342410Z] 11:57:04     INFO - #03: nsFontMetrics::nsFontMetrics [mfbt/RefPtr.h:53]
[task 2017-04-12T11:57:04.344022Z] 11:57:04     INFO - 
[task 2017-04-12T11:57:04.345656Z] 11:57:04     INFO - #04: nsFontCache::GetMetricsFor [mfbt/RefPtr.h:110]
```

...and so forth, where roughly half of the text is [task 2017-04-12T...] junk that's really not important.  I'm sure it's useful for some cases, but it would be nice if the log viewer automatically suppressed those bits, maybe with a toggle to turn them back on.

Bonus points for suppressing the test harness timestamps, or providing an option to--I know those can be useful by default.

**Comment by eliperelman** [comment]
The logviewer doesn't insert any timestamps, so these are included in the text of the log itself.

**Comment by froydnj** [comment]
> The logviewer doesn't insert any timestamps, so these are included in the text of the log itself.

Yes, I understand that.  I'm asking for an option for those to be suppressed in the log viewer.

**Comment by eliperelman** [comment]
Ah ok. Thinking about the implementation behind this, it would most likely include some sort of pre-processing function that would need to be executed against each line.
